migration/migrator/data/course_tables.sql

```diff
@@ -714,7 +714,12 @@ CREATE TABLE public.course_materials (
     hidden_from_students boolean,
     priority double precision NOT NULL,
     url text,
-    title character varying(255)
+    title character varying(255),
+    uploaded_by character varying(255) DEFAULT NULL::character varying,
+    uploaded_date timestamp with time zone,
+    last_edit_by character varying(255) DEFAULT NULL::character varying,
+    last_edit_date timestamp with time zone,
+    CONSTRAINT check_dates CHECK (((uploaded_date IS NULL) OR (last_edit_date IS NULL) OR (uploaded_date <= last_edit_date)))
```

Comment:
It might be better to strengthen the logic here?
```suggestion
    CONSTRAINT check_dates CHECK (
        (uploaded_date IS NOT NULL AND last_edit_date IS NULL) OR 
        (uploaded_date IS NULL AND last_edit_date IS NULL) OR 
        (uploaded_date IS NOT NULL AND uploaded_date <= last_edit_date)
    )
```